interfaces/api.interfaces.ts

```diff
@@ -54,31 +58,43 @@ export interface Dataset {
     record_count?: number;
     description?: string;
 }
+
 export interface Record {
     id?: number;
     client_id?: string;
     dataset?: number;
     site?: number | null;
-    source_info?: {[key: string]: string | number};
+    source_info?: { [key: string]: string | number };
     last_modified?: string;
     created?: string;
-    data?: {[key: string]: any} | null;
+    data?: { [key: string]: any } | null;
     published?: boolean,
     consumed?: boolean,
     datetime?: string;
     geometry?: GeoJSON.Point | GeoJSON.LineString | GeoJSON.MultiLineString | GeoJSON.Polygon | GeoJSON.MultiPolygon | null;
     species_name?: string;
     name_id?: number;
 }
+
 export interface RecordResponse {
     count: number;
     results?: Record[];
 }
+
+export interface Media {
+    id: number;
+    record: number
+    file: string;
+    created: string;
+    last_modified: string
+}
```

Comment:
I think you should add a `?` after the name. We had issues sometimes without it.